modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/inputs/StringInput.es.js

```diff
@@ -46,7 +65,7 @@ class StringInput extends React.Component {
 				disabled={disabled}
 				onChange={this._handleChange}
 				type="text"
-				value={value}
+				value={this.state.value}
```

Comment:
This is just two questions: Why is it better to have the unescaped value stored in the state? I see that you update this state in the onChange, was the prop not updating its value correctly before?